docs/source/Manuals/SManual/SManual.rst

```diff
@@ -1044,16 +1044,16 @@ the ``gasstation``.)
 
 
 .. _`simpydownload`: https://github.com/SimPyClassic/SimPyClassic
-.. _`Python web-site` : http://www.Python.org
+.. _`Python web-site` : https://www.Python.org
 
 
 .. .. _Top: Contents_
 
 .. _monitors: `Monitors and Recording Simulation Results`_
 
 
-.. _`Python random module`: http://www.python.org/doc/current/lib/module-random.html
-.. _Python: http://www.Python.org
+.. _`Python random module`: https://docs.python.org/3/library/random.html
+.. _Python: https://www.Python.org
```

Comment:
Capital p again

Reply:
:-( will grep.  I was relying on the link checker - it complained for some of them

Reply:
fixed